lessons/beginners/exceptions/index.md

```diff
@@ -182,54 +181,49 @@ finally:
 ```
 
 
-## Úkol
-
-Doplň do geometrické kalkulačky (nebo 1-D piškvorek, máš-li je) ošetření chyby,
-která nastane když uživatel nezadá číslo.
 
-{% filter solution %}
+## Vyvolání chyby
 
-Možné řešení pro geometrickou kalkulačku:
+Občas se stane, že výjimku budeš potřebovat vyvolat {{gnd('sám', 'sama')}}.
 
-```python
+Často se to stává když píšeš nějakou obecnou funkci.
+Třeba funkci na výpočet obsahu čtverce.
+Co se stane, když někdo zavolá `obsah_ctverce(-5)`?
 
-while True:
-    try:
-        strana = float(input('Zadej stranu čtverce v centimetrech: '))
-    except ValueError:
-        print('To nebylo číslo!')
-    else:
-        if strana <= 0:
-            print('To nedává smysl!')
-        else:
-            break
+* Zadal-li ono `-5` uživatel, je potřeba mu vynadat a zeptat se znovu.
+* Naměřil-li `-5` nějaký robotický aparát, je potřeba ho líp zkalibrovat.
+* Vyšel-li čtverec se stranou `-5` v nějakém výpočtu, je nejspíš potřeba opravit
```

Comment:
Reword without `-li`, masculine and (too strong) `vynadat` -> rather `upozornit`.

Reply:
I don't think `vynadat` is too strong here. I've used it quite successfully to refer to "bad user input" messages.

Reply:
Also about `-li`: I don't understand why that needs reqording.